package/models.py

```diff
@@ -1,13 +1,28 @@
 import os
-import inspect
+import shutil
 
 
 def display(snippet_name):
-    snippet_path = os.path.join(
-        os.path.dirname(__file__), "stash", f"{snippet_name}.py"
-    )
+    try:
+        snippet_path = os.path.join(
+            os.path.dirname(__file__), "stash", f"{snippet_name}.py"
+        )
 
-    with open(snippet_path, "r") as file:
-        source_code = file.read()
+        if os.path.isfile(snippet_path):
```

Comment:
Hey, since shutil is an advanced file operation, it should be useless to check ".isfile", better to remove it since shutil will already check

You can confirm it by using shutil without checking if file exists, and see if it errors out

Reply:
@Wahid7852 Okay Sure. I will do it.

Reply:
@Wahid7852 please check if it is okay now and let me know.